data/tips.cfg

```diff
@@ -264,13 +264,11 @@
 [tip]
     # Examples: oases, lava/volcanoes, illuminated caves
     text= _ "Some types of terrain give unique bonuses, such as healing or illumination."
-    # po: Translate the word "Flight" as in Wings of Victory
     source= _ "<i>― Galun, Flight Leader, 5YW</i>"
```

Comment:
This line should still have a translation hint. I suggest this would be clearer than the original:
```
    # po: Translate the word "Flight" as a collective noun for drakes
```

Reply:
Done.